src/services/configurations/index.js

```diff
@@ -4,11 +4,12 @@ import { checkStatus, parseJSON } from "../common";
 
 const config = {
     path: process.env.PUBLIC_URL + "/configurations/",
+    api: process.env.REACT_APP_API_URL,
```

Comment:
Can we please have a config.js file where all the configs are assigned. Accessing process.env directly is not a good idea. Also, have a default value set for each one of them